test/variants/parser/packing_concat01.v3

```diff
@@ -0,0 +1,2 @@
+//@parse
+packing MyScalar(a: 16, b: 16): 32 = a <<| b;
```

Comment:
Can you add some more negative parsing tests? Those are usually the ones that reveal parser bugs.